contracts/SmartSession.sol

```diff
@@ -378,13 +378,13 @@ contract SmartSession is ISmartSession, SmartSessionBase, SmartSessionERC7739 {
         view
         virtual
         override
-        returns (bool valid)
+        returns (bool)
     {
         bytes32 contentHash = string(contents).hashERC7739Content();
         PermissionId permissionId = PermissionId.wrap(bytes32(signature[0:32]));
         signature = signature[32:];
         if (!$enabledERC7739Content[permissionId].contains(msg.sender, contentHash)) return false;
-        valid = $erc1271Policies.checkERC1271({
+        bool valid = $erc1271Policies.checkERC1271({
```

Comment:
isn't this worse on gas?

Reply:
according to tests, it is increasing 4 units of gas

Reply:
thank you. I'll proceed with merging since it is more consistent

Reply:
thank you. I'll proceed with merging since it is more consistent